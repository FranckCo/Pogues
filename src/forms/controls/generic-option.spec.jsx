import React from 'react';
import renderer from 'react-test-renderer';

import GenericOption from './generic-option';

describe('Form controls - Generic option', () => {
  test('Should exists with the corresponding template', () => {
    const props = {
      value: 'Fake value',
      className: 'fake-class-name',
    };
    const tree = renderer
      .create(<GenericOption {...props}>Fake label</GenericOption>)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
