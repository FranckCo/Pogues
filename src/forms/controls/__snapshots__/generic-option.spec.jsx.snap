// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Form controls - Generic option > Should exists with the corresponding template 1`] = `
<div
  className="fake-class-name"
  data-value="Fake value"
>
  Fake label
</div>
`;
